### ISSUE TITLE
Getting Infinite error loop and Error 503 when I try to access on browser

### ISSUE THREAD

**Issue by iamvinny** [btcpayserver/btcpayserver-docker#2]
I have installed **docker-compose**, cloned this repository and then cd into the **Production** directory, once there I ran, **docker-compose up**, but it didn't work, I got stuck in a loop of errors. Sometimes it hangs for a few seconds in the output below, and then the error loop starts again..

If I try to access http://my_public_ip/ I get a _503 Service Temporarily Unavailable_ error.

I'm using Google Engine by the way.. I'm not sure if this docker is compatible with Azure only.

```
nginx-gen                            | 2018/01/05 19:46:50 Received event start for container cdca2f293225
btcpayserver_bitcoind                | 2018-01-05 19:46:50 Adding fixed seed nodes as DNS doesn't seem to be available.
letsencrypt-nginx-proxy-companion    | 2018/01/05 19:46:53 Debounce maxTimer fired
letsencrypt-nginx-proxy-companion    | 2018/01/05 19:46:53 Contents of /app/letsencrypt_service_data did not change. Skipping notification '/app/update_certs'
nginx-gen                            | 2018/01/05 19:46:55 Debounce minTimer fired
nginx-gen                            | 2018/01/05 19:46:55 Contents of /etc/nginx/conf.d/default.conf did not change. Skipping notification ''
```

**Comment by iamvinny** [comment]
What ports should be unblocked? Actually I have 443 and 80 open.

**Comment by iamvinny** [comment]
I also get the warning below when i run the `docker-compose up` command.

`WARNING: The BTCPAY_HOST variable is not set. Defaulting to a blank string.`

**Comment by NicolasDorier** [comment]
Have you set all env variable as described in https://github.com/btcpayserver/btcpayserver-docker/blob/master/Production/README.md ?


If yes, I suppose you are running `sudo docker-compose up`.
The problem with sudo is that it does not keep the env variable of your current session, and so they are not passed to the service.

```
sudo su -
docker-compose up
```

Might give you better result.

**Comment by iamvinny** [comment]
Thanks, setting the env variables solved the problem, now I got another problem with nbxplorer. I'll open another issue.
